packages/kbn-config-schema/package.json

```diff
@@ -6,7 +6,9 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "build": "../../node_modules/.bin/tsc",
-    "kbn:bootstrap": "yarn build"
+    "build": "yarn clean && ../../node_modules/.bin/tsc",
+    "kbn:bootstrap": "yarn build",
+    "kbn:watch": "yarn build --watch",
+    "clean": "rm -rf target"
```

Comment:
We can disable incremental builds then as tsbuildinfo file is stored in ` target/types` folder

Reply:
Is that what we want? The package will only be cleaned and built during bootstrap when there are changes.

Reply:
> when there are changes.

My point was that we should opt-out from TS incremental builds if we implement the cache invalidation policy manually.

Reply:
it means to add 
```json
  "compilerOptions": {
    "incremental": false
  },
```
to `packages/*/tsconfig.json`

Reply:
Done in ad731f265161b54f96f593b34d6145a102c76d3f for packages we are cleaning.